assets/gcp.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: gce-standard
```

Comment:
We should call this just `standard` to be inline with old openshift-ansible.

Reply:
This has been updated this to be `standard`.